ideascube/monitoring/templates/monitoring/loan.html

```diff
@@ -28,7 +28,7 @@
         </tr>
         {% for loan in loans %}
             <tr>
-                <td>{{ loan.specimen.item }} - {{ loan.specimen.barcode }}</td>
+                <td>{{ loan.specimen.item }} - {{ loan.specimen.user_identifier }}</td>
```

Comment:
If user start to scan the serial and do not print a specific ideascube barcode there will be no barcode and so, nothing displayed.
I think it's better to display the serial if there is no barcode as this is what the user use.